Output/pjrcUSB/arm/usb_dev.c

```diff
@@ -1132,7 +1132,7 @@ void usb_rx_memory( usb_packet_t *packet )
 }
 
 // Call whenever there's an action that may wake the host device
-void usb_resume()
+int usb_resume()
```

Comment:
To be consistent with the rest of the code base, please change to uint8_t.
(int is kinda scary when dealing with multiple microcontroller architectures...).

Otherwise it looks great! Thanks a lot.